<!-- run the real robo -->

<launch>
    <include file="$(find panda_moveit_config)/launch/franka_control.launch">
        <arg name="robot_ip" value="$(arg robot_ip)"/>
    </include>

</launch>
